app/src/routes/+page.svelte

```diff
@@ -11,6 +14,8 @@
     if (!selectedCategory) return true;
     return category.toLowerCase() === selectedCategory;
   });
+
+  $: amountOfVisibleEvents = 6;
```

Comment:
Perhaps `let` makes more sense in this context. The `$:` statement runs whenever the values it depends on change.